### ISSUE TITLE
setuptools 20.2.x changes _markerlib implementation, breaking the test scripts

### ISSUE THREAD

**Issue by rcrowder** [numenta/nupic#3036]
A user has reported an issue on the mailing list with a development installation of NuPIC running on Ubuntu 14.04 [1] It relates to changes made in the Python setuptools package (version 20.2.x) [2] and the _markerlib implementation. This breaks, for example, scripts/run_nupic_tests.py resulting in the following thrown exception -

> from _markerlib import compile as compile_marker
> ImportError: No module named _markerlib
> 

I'm unsure which package is throwing this, the mailing list email thread has more information of which Python packages the user has installed.

1 http://lists.numenta.org/pipermail/nupic_lists.numenta.org/2016-March/012979.html
2 https://pythonhosted.org/setuptools/history.html

**Comment by rcrowder** [comment]
Further information on the email thread shows that this looks more related to mismatching Python packages with changes in setuptools. I tried recreating this in Ubuntu 14.04 with up-to-date packages and failed to recreate the issue. A fresh Ubuntu install with the packages and certain versions installed would be required to see if this is an issue with, for example, a mismatch between py.test and a more recent setuptools package. Closing this issues for now, as it doesn't look related to NuPIC, but more related to a mix of upgraded and non-upgraded packages.
